pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.0.6</version>
+        <version>3.0.7</version>
```

Comment:
When changing versions, also remember to update the `DEPENDENCIES` file with the Eclipse Dash Tool:
```
mvn org.eclipse.dash:license-tool-plugin:license-check -Ddash.summary=DEPENDENCIES -Ddash.project=automotive.tractusx
```

Reply:
I have done this now and made an addtional commit to the current branch.